src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/FileBasedPrograms/VirtualProjectXmlProvider.cs

```diff
@@ -30,8 +93,8 @@ private static class Sha256Hasher
     {
         public static string Hash(string text)
         {
-            byte[] bytes = Encoding.UTF8.GetBytes(text);
-            byte[] hash = SHA256.HashData(bytes);
+            var bytes = Encoding.UTF8.GetBytes(text);
```

Comment:
I don't know why a bunch of locals just changed to `var`. Possibly I ran a refactoring without meaning to.